cypress/integration/computation_menu.spec.ts

```diff
@@ -36,7 +36,6 @@ function waitForLoad(iteration: number = 0) {
 
 describe("ComputationMenu", () => {
     beforeEach(() => {
-        cy.clearLocalStorage();
         cy.viewport(1080, 720);
```

Comment:
Why are we using 1080 x 720 viewports?

Reply:
Because the default size is so small the page turns to the mobile format.

Reply:
Hm, could it be a thought to test both sizes?

Reply:
I can make an issue.